lib/components/signUp.dart

```diff
@@ -184,7 +185,7 @@ class _SignUpScreenState extends State<SignUpScreen> {
                   ),
                 ),
                 SizedBox(
-                  height: 5.0,
+                  height:  MediaQuery.of(context).size.height * 0.0005,
```

Comment:
Just using MediaQuery's height wouldn't work in a wider aspect ration phone, we'd have to use breakpoints to assign heights, checkout [DevicePreivew](https://pub.dev/packages/device_preview) to test your code with different screen sized.